### PR TITLE
Avoid nameserver duplication

### PR DESCRIPTION
**Pull request by maneta**
correct `server == resolver` case in #630

### CODE REVIEW

File: spec/resty/resolver_spec.lua [hunk 1]
```diff
@@ -179,11 +179,12 @@ nameserver 127.0.0.1
     end)
 
     it('returns nameserver touples', function()
```
Comment (mikz) [inline spec/resty/resolver_spec.lua]: I think this deserves a new test.

File: t/resolver.t [hunk 2]
```diff
@@ -101,3 +101,29 @@ GET /t
...
+    }
+  }
+--- request
+GET /t
+--- response_body
+nameservers: 2 127.0.1.153 1.2.3.453
+--- user_files
+>>> resolv.conf
+nameserver 127.0.1.1
+nameserver 1.2.3.4
```
Comment (mikz) [inline t/resolver.t]: Missing newline at the end of the file. Would be good to set-up your editor to always keep a new-line at the EOF.

File: spec/resty/resolver_spec.lua (hunk 1 above)
Comment (davidor) [reply spec/resty/resolver_spec.lua]: I agree. Also, I think that we have similar problems with other tests. For example, this test is checking that commented lines are ignored, but there isn't an `it` clause explaining that or an explicit assertion.

Comment (maneta) [reply spec/resty/resolver_spec.lua]: done in e2755eb835e200bf682cd2720f74fe609cb4046d

File: t/resolver.t (hunk 2 above)
Comment (maneta) [reply t/resolver.t]: done in 3afea9860ca9d62d22ebd0a40f85ad312db82138

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Error loading policy chain configuration JSON with null value [PR #626](https://github.com/3scale/apicast/pull/626)
 - Splitted `resolv.conf` in lines,to avoid commented lines  [PR #618](https://github.com/3scale/apicast/pull/618)
+- Avoid `nameserver` repetion from `RESOLVER` variable and `resolv.conf` file [PR #636](https://github.com/3scale/apicast/pull/636)
```
Comment (davidor) [inline CHANGELOG.md]: typo

File: spec/resty/resolver_spec.lua [hunk 4]
```diff
@@ -182,10 +182,19 @@ nameserver 127.0.0.1
       local nameservers = resty_resolver.parse_nameservers(tmpname)
 
       assert.equal(2, #nameservers)
-      assert.same({ '127.0.0.2', 53 },  nameservers[1])
+      assert.same({ '127.0.0.2', 53 }, nameservers[1])
       assert.same({ '127.0.0.1', 53 }, nameservers[2])
     end)
 
+    it('do not replicates nameservers from resolver env var', function()
```
Comment (davidor) [inline spec/resty/resolver_spec.lua]: typo

**Comment by davidor** [comment]
:+1: 

Thanks for your contribution @maneta

**Pull request merged**